Zscaler/zscaler-zia/tests/test_event_dns.json

```diff
@@ -0,0 +1,40 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Zscaler Internet Access [BETA]",
+        "dialect_uuid": "dc0f339f-5dbe-4e68-9fa0-c63661820941"
+      }
+    },
+    "message": "{\"sourcetype\": \"zscalernss-dns\",\"event\": {\"datetime\": \"Mon Aug 28 08:05:55 2023\",\"user\": \"john.doe@example.orf\",\"department\": \"Financial%20Dept\",\"location\": \"Road%20Warrior\",\"reqaction\": \"Allow\",\"resaction\": \"Allow\",\"reqrulelabel\": \"DNS_1\",\"resrulelabel\": \"Zscaler Bypass Traffic\",\"dns_reqtype\": \"AAAA\",\"dns_req\": \"test.example.org\",\"dns_resp\": \"NotFound\",\"srv_dport\": \"53\",\"durationms\": \"0\",\"clt_sip\": \"1.2.3.4\",\"srv_dip\": \"5.6.7.8\",\"category\": \"Corporate Marketing\",\"respipcategory\": \"Other\",\"deviceowner\": \"johndoe\",\"devicehostname\": \"hostname\"}}"
+  },
+  "expected": {
+    "message": "{\"sourcetype\": \"zscalernss-dns\",\"event\": {\"datetime\": \"Mon Aug 28 08:05:55 2023\",\"user\": \"john.doe@example.orf\",\"department\": \"Financial%20Dept\",\"location\": \"Road%20Warrior\",\"reqaction\": \"Allow\",\"resaction\": \"Allow\",\"reqrulelabel\": \"DNS_1\",\"resrulelabel\": \"Zscaler Bypass Traffic\",\"dns_reqtype\": \"AAAA\",\"dns_req\": \"test.example.org\",\"dns_resp\": \"NotFound\",\"srv_dport\": \"53\",\"durationms\": \"0\",\"clt_sip\": \"1.2.3.4\",\"srv_dip\": \"5.6.7.8\",\"category\": \"Corporate Marketing\",\"respipcategory\": \"Other\",\"deviceowner\": \"johndoe\",\"devicehostname\": \"hostname\"}}",
```

Comment:
Can you extract:
`clt_sip` -> `source.ip`
`srv_dip` -> `destination.ip`
`srv_dport` -> `destination.port`
`dns_req` -> `dns.question.name`
`dns_resp` -> `dns.response_code`

and move
`dns.type` to `dns.question.type`
`user.name` to `user.email`
`zscaler.zia.device.hostname` to `host.name`